pkg/controller/account/account_controller.go

```diff
@@ -335,6 +346,45 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 			return reconcile.Result{}, err
 		}
 
+		// Create SRE IAM user
+		SREIAMUserSecret, err := r.BuildIAMUser(reqLogger, awsAssumedRoleClient, currentAcctInstance, iamUserNameSRE, request.Namespace)
```

Comment:
This var name is a bit confusing, is it for a user or a secret

Reply:
Its the SRE IAM user secret, I can change it thoughts?

Reply:
Updated the comment